script/Arcueid/Day 11/11_00_ARC11_3C.txt

```diff
@@ -340,7 +350,8 @@ Is this some kind of joke? What language is this?
 [sha:98ce82191f016cbab8df6b4c347dd44c2bfc6183]{
 -- Page 313, Offset 30436.
 -- 紙きれには、ただ、ばいばい、としか、書かれていなかった。
-Written on the paper, there's nothing else but 'goodbye'.
+// どこの国の言葉 doesn't really land unless we modify 'baibai' into 'sayonara' or whatever
```

Comment:
yeah, Manolo and I talked about this and I think we meant to ask the group but just forgot. I think putting "sayonara" is just fine